fastlane/Jetpack-Fastfile

```diff
@@ -140,6 +144,67 @@ lane :update_jetpack_appstore_strings do |options|
   )
 end
 
+#####################################################################################
+# download_jetpack_localized_metadata
+# -----------------------------------------------------------------------------------
+# Downloads localized metadata for App Store Connect from GlotPress.
+# -----------------------------------------------------------------------------------
+# Usage:
+# bundle exec fastlane download_localized_metadata
+#####################################################################################
+desc 'Downloads localized metadata for App Store Connect from GlotPress'
+lane :download_jetpack_localized_app_store_metadata do
+  # No need to `cd` into `fastlane` because of how Fastlane manages its paths
+  # internally.
+  sh './download_metadata.swift jetpack'
+
+  jetpack_metadata_glob = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'jetpack_metadata/**/*.txt')
+  sh "git add #{jetpack_metadata_glob}"
+  git_commit(
+    path: jetpack_metadata_glob,
+    message: 'Update Jetpack metadata translations'
+  )
+end
+
+#####################################################################################
+# update_jetpack_metadata_on_app_store_connect
+# -----------------------------------------------------------------------------------
+# This lane reads the information from `fastlane/jetpack_metadata/` and uploads it to
+# App Store Connect for the version specified in the version xcconfig file.
+#
+# Optionally, it can also upload new screenshots, overriding the existing ones.
+# -----------------------------------------------------------------------------------
+# Usage:
+# bundle exec fastlane update_jetpack_metadata_on_app_store_connect
+#
+# Example:
+# bundle exec fastlane update_jetpack_metadata_on_app_store_connect
+# bundle exec fastlane update_jetpack_metadata_on_app_store_connect with_screenshots:true
+#####################################################################################
+desc 'Uploads the Jetpack metadata to App Store Connect, localized, and optionally including screenshots.'
+lane :update_jetpack_metadata_on_app_store_connect do |options|
+  # Skip screenshots by default. The naming is "with" to make it clear that
+  # callers need to opt-in to adding screenshots. The naming of the deliver
+  # parameter, on the other hand, uses the skip verb.
+  with_screenshots = options.fetch(:with_screenshots, false)
+  skip_screenshots = with_screenshots == false
+
+  deliver(
+    app_identifier: APP_IDENTIFIER,
+    app_version: read_version_from_config,
```

Comment:
I think we should to this (reading the version from the `.xcconfig`) for all the iOS apps. It would remove the need to modify the `Deliverfile` during the code freeze.

Reply:
Good point and good idea! 👍